bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/ImportJob.java

```diff
@@ -109,6 +109,11 @@
     @SuppressWarnings("unused")
     void setSourcePattern(ValueProvider<String> sourcePath);
 
+    @Description("Optional BigTable write throttling in milliseconds")
+    ValueProvider<String> getBigtableWriteThrottleMs();
```

Comment:
"ValueProvider<Integer>"
Set default to 0

Reply:
ValueProvider\<Integer\>

Reply:
Generally, I don't like having one config/flag that depends another.
If you must, please add a comment that says that > 0 value here will enable throttling.